src/server/api/routers/login.ts

```diff
@@ -67,20 +71,73 @@ export const loginRouter = createTRPCRouter({
             },
             HTMLPart: resetTemplate(resetLink),
             Subject: "Hack Western 11 Password Reset",
-          }
-        ]
-      })
+          },
+        ],
+      });
       // TODO: send email with reset link
 
-      emailReq.then(( result ) => {
-        console.log(result);
-      })
-      .catch((err) => {
-        console.error(err);
-      });
+      emailReq
+        .then((result) => {
+          console.log(result);
+        })
+        .catch((err) => {
+          console.error(err);
+        });
 
       return {
         success: true,
       };
-  }),
+    }),
+});
+export const createUserRouter = createTRPCRouter({
+  create: publicProcedure
+    .input(z.object({ email: z.string(), password: z.string() }))
```

Comment:
We should make the input validator more specific here so we can get better errors:

```suggestion
    .input(z.object({
       email: z.string().email(),
       password: z.string().max(255)
    }))
```

Reply:
sorry this is being handled by Joys ticket #35